### PR TITLE
[VI] Add `Bit rate` and `Guides/Audio Editing`

### PR DESCRIPTION
**Pull request by TamamoNoMae13**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [ ] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

**Comment by TamamoNoMae13** [comment]
cc @huupoke12 @minhducsun2002 for reviewing.

File: wiki/Guides/Audio_Editing/vi.md [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+   1. Nhấp `Edit`
+   2. Nhấp `Paste`
+6. Phát lại cả bài và đảm bảo rằng đoạn lặp nghe thật hay.
+7. Lặp lại nếu cần thiết.
+8. Nhấn `Ctrl` + `Shift` + `E`, sau đó ở `Save as type:` chọn `MP3 Files`, hoặc:
+   1. Nhấp `File`, rồi `Export`, rồi `Export as MP3`.
+9. Trong phần `Format Options`, thay đổi các thiết lập như sau: 
+   1. `Bit Rate Mode`: `Preset`
+   2. `Quality`: `Medium, 145-185 kbps`
+10. Điều hướng tới nơi mà bạn muốn lưu tệp.
```
Comment (huupoke12) [inline wiki/Guides/Audio_Editing/vi.md:55]: Ditto

File: wiki/Guides/Audio_Editing/vi.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+   - Bạn cũng có thể đổi tên tệp.
+11. Nhấp chuột `Save` và một hộp thoại sẽ xuất hiện để bạn nhập siêu dữ liệu (metadata).
+12. Nhấp chuột `OK` sau khi nhập xong siêu dữ liệu.
```
Comment (huupoke12) [inline wiki/Guides/Audio_Editing/vi.md:58]: Ditto for 11 and 12

File: wiki/Guides/Audio_Editing/vi.md [hunk 3]
```diff
@@ -0,0 +1,147 @@
...
+   2. Nhấp `Save complete audio...`.
+9. Điều hướng tới nơi mà bạn muốn lưu tệp.
+   - Bạn cũng có thể đổi tên tệp.
+10. Nhấp `Save`.
+
+### Cắt
+
+Cài đặt và mở mp3DirectCut, sau đó làm theo các bước dưới đây:
+
+1. Open the `.mp3` file that you want to crop.
```
Comment (huupoke12) [inline wiki/Guides/Audio_Editing/vi.md:114]: Translation isn't finished.

File: wiki/Guides/Audio_Editing/vi.md [hunk 4]
```diff
@@ -0,0 +1,147 @@
...
+   - Bạn cũng có thể đổi tên tệp.
+5. Nhấp chuột `Save` và một hộp thoại sẽ xuất hiện để bạn nhập siêu dữ liệu (metadata).
+6. Nhấp chuột `OK` sau khi nhập xong siêu dữ liệu.
+
+### Lặp
+
+Cài đặt và mở Audacity, sau đó làm theo các bước dưới đây:
+
+1. Mở tệp `.mp3` mà bạn muốn chạy lặp.
+2. Kéo thả chuột để bôi đen phần bạn muốn lặp.
```
Comment (huupoke12) [inline wiki/Guides/Audio_Editing/vi.md:37]: Kéo thả ~~chuột~~

No mouse here as explained above.

File: wiki/Guides/Audio_Editing/vi.md [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+   - Bạn cũng có thể đổi tên tệp.
+11. Nhấp chuột `Save` và một hộp thoại sẽ xuất hiện để bạn nhập siêu dữ liệu (metadata).
+12. Nhấp chuột `OK` sau khi nhập xong siêu dữ liệu.
+
+### Cắt
+
+Cài đặt và mở Audacity, sau đó làm theo các bước dưới đây:
+
+1. Mở tệp `.mp3` mà bạn muốn cắt.
+2. Kéo thả chuột để bôi đen phần bạn muốn cắt.
```
Comment (huupoke12) [inline wiki/Guides/Audio_Editing/vi.md:65]: Ditto

File: wiki/Guides/Audio_Editing/vi.md [hunk 6]
```diff
@@ -0,0 +1,147 @@
...
+   - Đây sẽ là phần kết thúc (outro) dài ngoằng mà bạn không muốn map.
+3. Nhấn `Delete`.
+4. Kéo thả chuột khoảng 3 đến 5 giây cho tới cuối bài.
```
Comment (huupoke12) [inline wiki/Guides/Audio_Editing/vi.md:68]: Ditto

File: wiki/Guides/Audio_Editing/vi.md [hunk 7]
```diff
@@ -0,0 +1,147 @@
...
+6. Nhấp `Fade Out`.
+7. Nhấn `Ctrl` + `Shift` + `E`, sau đó ở `Save as type:` chọn `MP3 Files`, or:
+   1. Nhấp `File`, rồi `Export`, rồi `Export as MP3`.
+8. Trong phần `Format Options`, thay đổi các thiết lập như sau: 
+   1. `Bit Rate Mode`: `Preset`
+   2. `Quality`: `Medium, 145-185 kbps`
+9. Điều hướng tới nơi mà bạn muốn lưu tệp.
+   - Bạn cũng có thể đổi tên tệp.
+10. Nhấp chuột `Save` và một hộp thoại sẽ xuất hiện để bạn nhập siêu dữ liệu (metadata).
+11. Nhấp chuột `OK` sau khi nhập xong siêu dữ liệu.
```
Comment (huupoke12) [inline wiki/Guides/Audio_Editing/vi.md:79]: Ditto for 10 and 11

File: wiki/Guides/Audio_Editing/vi.md [hunk 8]
```diff
@@ -0,0 +1,147 @@
...
+3. Nhấn `Delete`.
+4. Kéo thả chuột khoảng 3 đến 5 giây cho tới cuối bài.
+5. Nhấp `Effect`.
+6. Nhấp `Fade Out`.
+7. Nhấn `Ctrl` + `Shift` + `E`, sau đó ở `Save as type:` chọn `MP3 Files`, or:
+   1. Nhấp `File`, rồi `Export`, rồi `Export as MP3`.
+8. Trong phần `Format Options`, thay đổi các thiết lập như sau: 
+   1. `Bit Rate Mode`: `Preset`
+   2. `Quality`: `Medium, 145-185 kbps`
+9. Điều hướng tới nơi mà bạn muốn lưu tệp.
```
Comment (huupoke12) [inline wiki/Guides/Audio_Editing/vi.md:76]: Ditto

File: wiki/Guides/Audio_Editing/vi.md [hunk 9]
```diff
@@ -0,0 +1,147 @@
...
+   - Bạn cũng có thể đổi tên tệp.
+10. Nhấp chuột `Save` và một hộp thoại sẽ xuất hiện để bạn nhập siêu dữ liệu (metadata).
+11. Nhấp chuột `OK` sau khi nhập xong siêu dữ liệu.
+
+## mp3DirectCut
+
+[mp3DirectCut](https://mpesch3.de) là phần mề chỉnh sửa âm thanh miễn phí, có thể sửa tệp MP3 mà không cần mã hoá lại, thường sẽ ngăn ngừa việc giảm chất lượng. Đây là phần mềm được khuyên dùng khi cần nâng/hạ âm lượng hoặc cắt âm thanh.
```
Comment (huupoke12) [inline wiki/Guides/Audio_Editing/vi.md:83]: phần *mềm*

Comment (huupoke12) [inline wiki/Guides/Audio_Editing/vi.md:83]: Đây là khuyến nghị khi cần tăng/giảm âm lượng hoặc cắt âm thanh.

The guide doesn't recommend the software, it recommends avoid re-encoding when needing to adjust volume or cropping.

File: wiki/Guides/Audio_Editing/vi.md [hunk 10]
```diff
@@ -0,0 +1,147 @@
...
+
+### Lặp
```
Comment (huupoke12) [inline wiki/Guides/Audio_Editing/vi.md:85]: Everything is the same as above, including my reviews.

**Review by huupoke12 — CHANGES_REQUESTED** [review]

**Comment by huupoke12** [comment]
I feel this guide shouldn't exist, most of the content is just general software manual, which should be referenced by this article. Things like Ctrl+C, Ctrl+V, Drag and drop are generic PC shortcuts, the reader should already be familiar these things if they want to make beatmaps, which is a much more difficult task.

**Comment by TamamoNoMae13** [comment]
In most cases, I'd love to rewrite the English guide more.

**Comment by TamamoNoMae13** [comment]
> I think this guide shouldn't exist, most of the content is just general software manual, which should be referenced by this article. Things like Ctrl+C, Ctrl+V, Drag and drop are generic PC shortcuts, the reader should already be familiar with these things if they want to make beatmaps, which is a much more difficult task.

**Comment by TamamoNoMae13** [comment]
> I think this guide shouldn't exist, most of the content is just general software manual, which should be referenced by this article. Things like Ctrl+C, Ctrl+V, Drag and drop are generic PC shortcuts, the reader should already be familiar with these things if they want to make beatmaps, which is a much more difficult task.

In most cases, I'd love to rewrite the English guide more than doing this.

**Pull request closed without merging**